### PR TITLE
Hotfix to start at false instead of true

### PR DESCRIPTION
**Pull request by LuqDragon**

### CODE REVIEW

File: src/scripts/pokedex/PokedexHelper.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import TypeColor = GameConstants.TypeColor;
 
 class PokedexHelper {
-    public static toggleStatisticShiny = ko.observable(true);
+    public static toggleStatisticShiny = ko.observable(false);
```
Comment (RedSparr0w) [inline src/scripts/pokedex/PokedexHelper.ts:4]: Wouldn't this make it show the non-shiny image on the individual Pokémon modal?

File: src/components/pokedex.html [hunk 2]
```diff
@@ -88,7 +88,7 @@ <h4 style="margin-bottom: 0">Pokédex</h4>
                        <div class="col-sm-2">
                         <label class="form-check-label">
                             <input class="form-check-input" type="checkbox" id="pokedex-filter-held-item"
-                                   value="false" checked onclick="PokedexHelper.toggleAllShiny(!PokedexHelper.toggleAllShiny())"> Hide shiny image
+                                   value="true" onclick="PokedexHelper.toggleAllShiny(!PokedexHelper.toggleAllShiny())"> Hide shiny image
```
Comment (RedSparr0w) [inline src/components/pokedex.html:91]: I've also just pushed this, as I forgot to remove the `checked` attribute when I flipped the setting name.

File: src/scripts/pokedex/PokedexHelper.ts (hunk 1 above)
Comment (LuqDragon) [reply src/scripts/pokedex/PokedexHelper.ts:4]: Oh god, my head is exploding this days, sorry! I will push the right tested things

File: src/components/pokedex.html (hunk 2 above)
Comment (LuqDragon) [reply src/components/pokedex.html:91]: I removed the checked and keep `false`, this `value` isnt used and for me represent if checkbox is checked or not

**Review by RedSparr0w — APPROVED** [review]
👍 Looks good, Thanks for the PR!

**Pull request merged**